ibis/pandas/tests/test_udf.py

```diff
@@ -225,49 +250,107 @@ def my_corr2(lhs, **kwargs):
             pass
 
 
-def test_compose_udfs():
+def test_compose_udfs(t2, df2):
```

Comment:
what happens if the udaf raises an exception? are these caught in a reasonable way? just asking for a test (not even sure what this should do)

Reply:
Added `test_udf_error`